<center>
    <img src="https://s3-api.us-geo.objectstorage.softlayer.net/cf-courses-data/CognitiveClass/Logos/organization_logo/organization_logo.png" width="300" alt="cognitiveclass.ai logo"  />
</center>


#### Add your code below following the instructions given in the course


<a href="https://dataplatform.cloud.ibm.com/gallery">IBM Cloud Gallery</a> 

# My Jupyter Notebook on IBM Watson Studio

##### PIYUSH GUPTA
I want to be a Data Analyst.

*It is because I like to derive insights from the large amount of data for the organizations in order to help in their business growth.*



### Following program derives the area of circle

In [1]:
def area(r):
    return 3.14*r*r

print("ENTER THE RADIUS")
radius=int(input())
a=area(radius)
print("CIRCLE OF RADIUS {0} HAVE AREA OF {1}".format(radius,a))

ENTER THE RADIUS


 10


CIRCLE OF RADIUS 10 HAVE AREA OF 314.0


| **NAME**   | **RADIUS(in metres)** | **AREA(in sq.metres)**  | 
|------------|-----------------------|-------------------------|
| **PAUL**   | 10                    | 314                     |
| **ARTHUR** | 25                    | 1962.5                  |
| **ROMY**   | 18                    | 1017.36                 |


[About Coursera](https://about.coursera.org/)

![Image](https://external-content.duckduckgo.com/iu/?u=https%3A%2F%2Ftse1.mm.bing.net%2Fth%3Fid%3DOIP.avxhtQqr5FoubGjq4sIXugHaEo%26pid%3DApi&f=1/600/400)